Add common cmd+k use cases (#26271)
DataDog/documentation: content/en/logs/_index.md

Start exploring your ingested logs in the [Log Explorer][1].

> **Tip**: To get to the Log Explorer quickly, press <kbd>Cmd</kbd>/<kbd>Ctrl</kbd> + <kbd>K</kbd> to open Datadog's global search, then search for `logs`.

Reviewer: ```suggestion
**Tip**: To open the Log Explorer from Datadog's global search, press <kbd>Cmd</kbd>/<kbd>Ctrl</kbd> + <kbd>K</kbd> and search for `logs`.
```

Reviewer: just suggesting a different phrasing. Feel free to ignore it or use it elsewhere.
